commit-event-service/src/test/scala/io/renku/commiteventservice/events/categories/commitsync/eventgeneration/LatestCommitFinderSpec.scala

```diff
@@ -48,103 +54,136 @@ class LatestCommitFinderSpec
 
   "findLatestCommit" should {
 
-    "return latest Commit info if remote responds with OK and valid body" in new TestCase {
+    "return latest Commit info if remote responds with OK and valid body - personal access token case" in new TestCase {
+      val maybePersonalAccessToken = personalAccessTokens.generateOption
```

Comment:
As the gitLabClient takes care of using the access token, I think we could simply generate some access token instead of personal and probably remove the next test.